server/belga/io/feed_parsers/belga_ansa.py

```diff
@@ -216,6 +216,7 @@ def parse_content(self, xml):
     def remove_prefix(self, text):
```

Comment:
would be good to have type annotation or something so this would actually not pass the CI